docs/buildRTD.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+# ------------------------------------------------------------
+# Script for RTD build(used for notebooks execution & doxygen)
+# ------------------------------------------------------------
+set -e
+
+BUILD_DIR=../buildRTD
+
+rm -rf $BUILD_DIR
+mkdir  $BUILD_DIR
+cd $BUILD_DIR
+
+cmake -DNRN_ENABLE_MPI=OFF -DNRN_ENABLE_INTERVIEWS=OFF ..
+make notebooks
+make doxygen
```

Comment:
I like having the newline at the end of the last line. Maybe I'm the only person in the world that uses an editor which would otherwise remove that unterminated line.  (same with next file)

Reply:
I'm using CLion and rarely notice/look at these kind of aspects :)